src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -10,4 +10,5 @@
   </div>
   <div class="spacing"></div>
   <a class="auth-button" href="${authUrl}">${authLabel}</a>
+  <a class="setting-button" href="/user">Setting</a>
```

Comment:
This settings button appears to the left of the auth button on the homepage but squished to the right of the auth button on the SRP so it's not consistent. 

Also, I think the setting button should only appear if the user is logged in? Settings page should probably not be accessible to non logged in users.

Reply:
this kind of fixed now, but see #88